### ISSUE TITLE
Building 1.8.7-p375 fails due to the absence of SVN repos

### ISSUE THREAD

**Issue by yoshinari-nomura** [rbenv/ruby-build#2119]
1.8.7-p375 is supposed to be fetched from the svn.ruby-lang.org:
  https://github.com/rbenv/ruby-build/blob/master/share/ruby-build/1.8.7-p375
However, svn.ruby-lang.org seems to be closed.

1.8.7-p374 is successfull because it has the tarball at ruby-lang.org.

### Steps to reproduce the behavior
```
LANG=C rbenv install --verbose 1.8.7-p375
```
### Expected vs. actual behavior
1.8.7-p375 is supposed to be fetched from the svn.ruby-lang.org, but failed.

### Logs
```
/tmp/ruby-build.20221228133048.1919217.vQ8zKM ~/prj/private/nenga/2023
Checking out http://svn.ruby-lang.org/repos/ruby/branches/ruby_1_8_7...
Redirecting to URL 'https://git.ruby-lang.org/repos/ruby/branches/ruby_1_8_7':
svn: E170013: Unable to connect to a repository at URL 'https://git.ruby-lang.org/repos/ruby/branches/ruby_1_8_7'
svn: E160013: '/repos/ruby/branches/ruby_1_8_7' path not found

BUILD FAILED (Ubuntu 20.04 using ruby-build 20221225-4-gfa46d54)
```

**Comment by hsbt** [comment]
Thanks. I fixed the repository URL for about this at #2120

**Comment by yoshinari-nomura** [comment]
Super-fast! thanks!
